fedimint-client/src/transaction/builder.rs

```diff
@@ -467,6 +485,8 @@ impl TransactionBuilder {
                             IdxRange::from_inclusive(input_idxs.clone()).expect("can't overflow"),
                         ))
                     } else {
+                        // TODO: In the future `InputBundle` and `OutputBundle` should make empty
+                        // bundles impossible, so this should not be possible
```

Comment:
Empty bundles being impossible is ok, but inputless txns are an explicit feature that we use in e.g. the LN module.

Reply:
Yeap. Inputless tx is just a tx without any bundles.